### PR TITLE
Documentation moving to github with improvements for "Github" authenticator

### PR DESCRIPTION
**Pull request by nalakas**
move the documentation to github

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+* Either Download the artifacts for this authenticator from [the store](https://store.wso2.com/store/assets/isconnector/details/bfed96a9-0d79-4770-9c55-22378d3a2812) or [build the project](Building From the Source) to get the jar and by adding it in the <IS-Home>/repository/components/dropins directory.
+
+* If you want to upgrade the Github Authenticator (.jar) in your existing IS pack, follow the bellow instructions.
+  * Stop WSO2 Identity Server if the server is already running.
+  * Download and extract the latest version of the connector artifacts (.jar, .war, gadgets etc.,) from the connector store.
+  * Replace the old .jar file found in the <IS_HOME>/repository/components/dropins folder with the new .jar file that you downloaded.
+
+## Building From the Source
+
+Follow the steps given below to build the Salesforce EI Inbound Endpoint from the source code:
```
Comment (menakaj) [inline docs/README.md]: Not Salesforce EI Inbound Endpoint. Its Github authenticator.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+* [Deploying Github artifacts](#Deploying-Github-artifacts)
+
+* [Configuring the Github App](#Configuring-the-Github-App)
+
+* [Deploying travelocity.com sample app](#Deploying-travelocity.com-sample-app)
+
+* [Configuring the identity provider](#Configuring-the-identity-provider)
+
+* [Configuring the service provider](Configuring the service provider)
```
Comment (menakaj) [inline docs/README.md]: The link is missing.

File: docs/README.md (hunk 1 above)
Comment (nalakas) [reply docs/README.md]: fixed

File: docs/README.md (hunk 2 above)
Comment (nalakas) [reply docs/README.md]: link added in under the sections

**Pull request merged**